src/Microsoft.Security.Utilities.Cli/ExportDetectionsCommand.cs

```diff
@@ -24,7 +24,7 @@ internal int Run(ExportDetectionsOptions options)
                      "PreciselyClassifiedSecurityKeys.json",
                       WellKnownRegexPatterns.PreciselyClassifiedSecurityKeys);
 
-            foreach (var precision in new[] { DetectionMetadata.HighConfidence, DetectionMetadata.MediumConfidence, DetectionMetadata.LowConfidence })
+            foreach (DetectionMetadata precision in new[] { DetectionMetadata.HighConfidence, DetectionMetadata.MediumConfidence, DetectionMetadata.LowConfidence })
```

Comment:
>DetectionMetadata [](http://example.com/codeflow?start=21&length=17)

i'm a very difficult person, but to me the type information in this data is clear because the enum value is encoded everywhere.

i'm pretty confident no set of style settings will satisfy my own list of do's and don'ts and automating this is a key value.